src/development/tools/devtools/release-notes/release-notes-2.17.0-src.md

```diff
@@ -8,6 +8,12 @@ towards some new features that are coming soon!
 See the full list of commits for DevTools 2.17.0
 [in the changelog](https://github.com/flutter/devtools/blob/master/CHANGELOG.md#2170).
 
+## Widget Inspector Updates
+* Added multi-package support for the root directory;[#4192](https://github.com/flutter/devtools/pull/4192)
```

Comment:
* Added support for manually setting the package directories for your app. If you've ever loaded the Inspector and noticed that some of your widgets are not present in the Widget tree, this may be an indication that the package directories for your app have not been set or detected properly. Your package directories determine which widgets the Inspector considers to be from "your" application. If you see an empty Inspector widget tree, or if you develop widgets across multiple packages and want widgets from all these locations to show up in your  tree, check the Inspector Settings dialog to ensure that your package directories are properly configured.

then add a screenshot of this setting